modules/exploits/multi/http/tomcat_mgr_upload.rb

```diff
@@ -210,6 +210,19 @@ def vars_get
     return vars
   end
 
+  def vars_undeploy
+    vars = {}
+    unless @csrf_token.nil?
+      vars = {
+        "path" => "/" + @app_base,
```

Comment:
For Metasploit testiners:

This looks like the only change vs `#vars_get`. Let's test whether or not the `@app_base` should actually be processed by our `target_uri` method.

Reply:
I've tested the change, and it deletes the app after deploy.
The value that gets generated with `"/" + @app_base` is a POST request parameter, and isn't actually included in the POST request URI. Hopefully that answers the question, but if not then feel free to @ me and I'll give it another test.
![image](https://user-images.githubusercontent.com/85949464/134023697-0ceaade0-6c9c-43d4-a680-5eba3fb73e55.png)